examples/plugins/afk.js

```diff
@@ -0,0 +1,50 @@
+// Plugins are loaded using the bot.loadPlugin() function
+// The load plugin function excepts one function as its only parameter
+// The plugin runs the function passing it two parameter
+// The former is the bot object and the latter is the bot option
+// the options contain version player info and other plugins
+// Here is an example AFK plugin
+
+// Each plugin should have a inject function
+// This is the function that will be put in bot.loadPlugin
+function inject (bot, option) {
+// It is a good idea to make a namespace for your plugin
```

Comment:
thats not what a namespace is, you are talking about scope, namespace doesnt exist in js